plugins/hybrid_ae_pkg/hybrid_ae_templates/hybrid_ae_template.html

```diff
@@ -0,0 +1,51 @@
+{% import 'forms.html' as forms %}
+
+<html>
+
+<head>
+    <link rel="stylesheet" type="text/css" href="{{url_for('static', filename='forms.css')}}" media="screen" />
+
+    <style type="text/css">
+        @import url("{{url_for('static', filename='forms.css')}}");
+    </style>
+</head>
+
+<body>
+    <div class="qhana-microfrontend">
+
+        <details class="qhana-help">
+            <summary>Help</summary>
+
+            Example Inputs:
+            <dl>
+                <dt>Input Data</dt>
+                <dd>A URL to the data input csv file. <code class="example">data:text/plain,0,0,0,0,0,0,0,0,0,0</code>
+                </dd>
+                <dt>Number of Qubits</dt>
+                <dd>The total number of qubits used for the quantum-neural-network. <code class="example">3</code></dd>
+                <dt>Embedding Size</dt>
+                <dd>The dimensionality of the embedding. <code class="example">2</code></dd>
+                <dt>QNN Name</dt>
+                <dd>The name of the quantum-neural-network to use as the autoencoder. <code class="example">QNN3</code>
+                </dd>
+                <dt>Training Steps</dt>
+                <dd>The number of training steps to train the autoencoder. <code class="example">100</code></dd>
+                <dl>
+
+                    <a href="{{example_values}}">Fill in example values.</a>
+
+                    <!-- TODO: Detailed description of the form options.-->
```

Comment:
It may be possible to use the simple template if the help text is moved into the model.
We could also allow the simple template to optionally display a help text (that uses markdown).

Reply:
Yes that makes more sense. Moved to simple template in the next commit.